x-pack/docs/en/security/securing-communications/security-minimal-setup.asciidoc

```diff
@@ -9,6 +9,8 @@ passwords for built-in users. You can add more users later, but using the
 built-in users simplifies the process of enabling security for your
 cluster.
 
+include::../configuring-stack-security.asciidoc[tag=minimal-security-note]
+
```

Comment:
I think we risk confusing people here.

The "basic security" instructions start by saying

> Complete the steps in Minimal security for the Elastic Stack 

That is, we expect production use cases to follow the minimal setup instructions, _and then keep going_ to the TLS setup instructions.

But, including this note says that the minimal instructions are not for production usage.
So, I expect that there will be people who are trying to setup production security and feel as though the basic page is saying "follow the minimal steps" but the minimal page says "don't do this".

I'm not sure on the right solution. Perhaps it's just a case of changing the wording of the note to say that the minimal is _insufficient_ for production cluster, rather than "not meant for production mode clusters".

Reply:
Good callout @tvernum. I also think we should remove the line to "Only use this security configuration for local development". As you mentioned, users need to complete the setup for minimal security before configuring TLS, so that line could be confusing. 

I like your suggested wording about minimal security being insufficient. I think that this revision might be more clear, and also inform users that they must enable minimal security before configuring TLS:
> The minimal security scenario is not sufficient for production mode clusters. If your cluster has multiple nodes, then you must enable minimal security and then configure Transport Layer Security (TLS) between nodes.